doc_src/index.rst

```diff
@@ -93,1883 +99,6 @@ For a script written in another language, just replace ``/bin/bash`` with the in
-
-By pressing :kbd:`Alt`\ +\ :kbd:`↑` and :kbd:`Alt`\ +\ :kbd:`↓`, a history search is also performed, but instead of searching for a complete commandline, each commandline is broken into separate elements just like it would be before execution, and the history is searched for an element matching that under the cursor.
-
-History searches are case-insensitive unless the search string contains an uppercase character, and they can be aborted by pressing the escape key.
-
-Prefixing the commandline with a space will prevent the entire line from being stored in the history.
-
-The command history is stored in the file ``~/.local/share/fish/fish_history`` (or
-``$XDG_DATA_HOME/fish/fish_history`` if that variable is set) by default. However, you can set the
-``fish_history`` environment variable to change the name of the history session (resulting in a
-``<session>_history`` file); both before starting the shell and while the shell is running.
-
-See the :ref:`history <cmd-history>` command for other manipulations.
-
-Examples:
-
-To search for previous entries containing the word 'make', type ``make`` in the console and press the up key.
-
-If the commandline reads ``cd m``, place the cursor over the ``m`` character and press :kbd:`Alt`\ +\ :kbd:`↑` to search for previously typed words containing 'm'.
-
-Navigating directories
-======================
-
-.. _directory-history:
-
-The current working directory can be displayed with the :ref:`pwd <cmd-pwd>` command, or the ``$PWD`` :ref:`special variable <variables-special>`.
-
-Directory history
------------------
-
-Fish automatically keeps a trail of the recent visited directories with :ref:`cd <cmd-cd>` by storing this history in the ``dirprev`` and ``dirnext`` variables.
-
-Several commands are provided to interact with this directory history:
-
-- :ref:`dirh <cmd-dirh>` prints the history
-- :ref:`cdh <cmd-cdh>` displays a prompt to quickly navigate the history
-- :ref:`prevd <cmd-prevd>` moves backward through the history. It is bound to :kbd:`Alt`\ +\ :kbd:`←`
-- :ref:`nextd <cmd-nextd>` moves forward through the history. It is bound to :kbd:`Alt`\ +\ :kbd:`→`
-
-.. _directory-stack:
-
-Directory stack
----------------
-
-Another set of commands, usually also available in other shells like bash, deal with the directory stack. Stack handling is not automatic and needs explicit calls of the following commands:
-
-- :ref:`dirs <cmd-dirs>` prints the stack
-- :ref:`pushd <cmd-pushd>` adds a directory on top of the stack and makes it the current working directory
-- :ref:`popd <cmd-popd>` removes the directory on top of the stack and changes the current working directory
-
 .. _initialization:
 
 Initialization files
```

Comment:
This means that after "installation and Start" we have sections
- Initialization files
- Future feature flags
- Event handlers
- Debugging fish scripts

which are mostly for advanced users. Previously, we had some light introduction, showcasing how to use `man` and so on.

I know that this is supposed to be reference documentation, not a tutorial (we have one), but maybe make the main page minimal, and move the details somewhere else.

maybe

- Initialization files -> language
- Future feature flags -> language
- Event handlers -> interactive (?)
- Debugging fish scripts -> interactive/language

Then the "Other help pages" can probably be weeded out a bit, since we mention most sections already.